docs/src/development/local/docksal.md

```diff
@@ -1,128 +1,200 @@
-This is an API Token found with your Platform.sh account
-and can be generated by going to the [API Tokens](https://accounts.platform.sh/user/api-tokens) page
-and clicking the `Create API Token` link.
-This enables you to interact with your Platform.sh account from within the CLI container.
+## 2. Create a projects directory
 
-```bash
-fin config set --global SECRET_PLATFORMSH_CLI_TOKEN=XXX
-```
+Create a directory to store all of your Docksal projects.
+It's recommended to be `~/Projects` on Linux/macOS and `C:\Projects` on Windows.
+See all [restrictions on the projects directory](https://docs.docksal.io/getting-started/project-setup/#projects-directory).
+
+## 3. Add an API token
+
+To connect Docksal with your Platform.sh account, use a Platform.sh API token.
+
+1. [Create an API token](./../cli/api-tokens.md#get-a-token) in the console.
+2. Add the token to your Docksal configuration by running this command:
 
-## Pulling a Platform.sh project
+   ```bash
+   fin config set --global SECRET_PLATFORMSH_CLI_TOKEN="<API_TOKEN>"
+   ```
 
-The Docksal CLI ships with the Platform.sh CLI tool.
-To use the tool and pull a project locally, make sure you have uploaded your SSH key to your Platform.sh account.
-Once that's done and a `SECRET_PLATFORMSH_CLI_TOKEN` has been added using the above step,
-you can set up your project with the following instructions.
+Now you can run `fin platform <command>` from the your computer without needing to install the Platform.sh CLI.
 
-Note: Replace `<PROJECT_ID>` with your project's ID, which can found within the Platform.sh dashboard.
-Replace `<PROJECT_DIRECTORY>` with the name of the local directory you'd like the project cloned into.
-Replace `<BRANCH_NAME>` with the name of the environment you're working on.
+## 4. Get your project
 
-If you do not already have Platform.sh CLI installed locally, you can use the one in the CLI image.
-The advantage of this would mean that the tool would never have to be installed locally and therefore is one less dependency.
+To get your project, pull it with the following command:
 
 ```bash
-fin run-cli 'platform get <PROJECT_ID> -e <BRANCH_NAME> <PROJECT_DIRECTORY>'
+fin pull init --hosting-platform='platformsh' --hosting-site=<PROJECT_ID> --hosting-env=<ENVIRONMENT_NAME> <TARGET_DIRECTORY_NAME>
 ```
 
-If you already have Platform.sh CLI installed locally, you can use that instead.
+This creates a directory with the specified name with all your files and code.
+It also adds a `.docksal` directory with all necessary Docksal configuration.
+These files are ignored by Platform.sh.
+
+## 5. Add commands
+
+Docksal doesn't automatically copy over any commands you have in your [build flavor](../../administration/../configuration/app/app-reference.md#build)
+and [hooks](../../configuration/app/hooks/_index.md).
+To get you project running like on Platform.sh, you have to add the commands to Docksal.
```

Comment:
⛏️ Second repetition of "like" in line 63, maybe adding a few more words sounds nicer?
```suggestion
To get your project running in the same way as on Platform.sh, you have to add the commands to Docksal.
```

Reply:
I think having a conjunction like "like" a couple times in separate paragraphs isn't so bad, but the second one seems easier to change without making it as long as "in the same way as".

Reply:
😍 The last change is much better than my proposal.